presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/Matcher.java

```diff
@@ -67,7 +68,8 @@ public interface Matcher
      * @param session The session information for the query
      * @param metadata The metadata for the query
      * @param symbolAliases The SymbolAliases containing aliases from the nodes sources
+     * @param cteNameMapping The one to one mapping between cte name in plan pattern and the cte name got from result query plan
```

Comment:
the one to one mapping between the name of the common table expression in the plan pattern and the name of the common table expression in the result query plan